tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/04_quantityDiscountsBasedOn.js

```diff
@@ -0,0 +1,182 @@
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_productSettings_quantityDiscountsBasedOn';
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ProductSettingsPage = require('@pages/BO/shopParameters/productSettings');
+const ProductsPage = require('@pages/BO/catalog/products');
+const AddProductPage = require('@pages/BO/catalog/products/add');
+const FOProductPage = require('@pages/FO/product');
+const CartPage = require('@pages/FO/cart');
+// Importing data
+const ProductFaker = require('@data/faker/product');
+
+let browser;
+let page;
+const productWithCombinations = new ProductFaker({
+  type: 'Standard product',
+  productHasCombinations: true,
+  price: 20,
+  combinations: {
+    Color: ['White', 'Black'],
+    Size: ['S'],
+  },
+  quantity: 10,
+  specificPrice: {
+    combinations: 'Size - S, Color - White',
+    discount: 50,
+    startingAt: 2,
+    reductionType: '%',
+  },
+});
+const firstAttributeToChoose = {color: 'White'};
+const secondAttributeToChoose = {color: 'Black'};
+const firstCartTotalTTC = 30;
+const secondCartTotalTTC = 40;
+let numberOfProducts = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    productSettingsPage: new ProductSettingsPage(page),
+    productsPage: new ProductsPage(page),
+    addProductPage: new AddProductPage(page),
+    foProductPage: new FOProductPage(page),
+    cartPage: new CartPage(page),
+  };
+};
+
+/*
+Choose quantity discounts based on 'Products'
+Create product with combinations and add a specific price(discount 50% for the first combination)
+Add the combinations to the cart and check the price TTC
+Choose quantity discounts based on 'Combinations'
+Check the cart price TTC
+ */
+describe('Choose quantity discount based on', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO and go to product settings page
+  loginCommon.loginBO();
+
+  it('should go to \'Shop parameters > Product Settings\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToProductSettingsPage1', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.shopParametersParentLink,
+      this.pageObjects.boBasePage.productSettingsLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
```

Comment:
@boubkerbribri To remove ?

Reply:
no, it's the first one in the test, we need it, maybe after adding [this](https://github.com/PrestaShop/PrestaShop/issues/17517), we can delete all of them from tests

Reply:
Thanks ;)